### ISSUE TITLE
Updated Switch: Focus indicator doesn't work on IE11

### ISSUE THREAD

**Issue by rlfriedman** [material-components/material-components-web#3212]
In the updated version of switch that uses ripple for state, the focus indicator doesn't show up when switches are focused on IE11. This is because IE11 doesn't run the ripple JS logic, so something might need to be changed inside ripple to still apply the focus/blur styling.

**Comment by kfranqueiro** [comment]
There are multiple components that don't get a ripple focus state in IE due to all ripple JS logic being short-circuited and due to having complex DOM where the focused element is a child of the ripple element rather than the same element:

* Select
* Switch
* Tab
* Text Field

Checkbox and Radio are also notable in that they _would_ have this problem, except that there is already separate CSS set up for no-ripple-JS cases (and that CSS has been there for easily over a year).

This problem is much more noticeable for switch (especially compared to select and text field) since there's no other affordance suggesting focus state.

I'm thinking we should be able to skip only the press ripple logic in IE while still getting the focus/blur logic, but this may require distinguishing via CSS whether the ripple is fully upgraded or only for focus/blur to ensure that other styles meant for upgraded ripple activation/deactivation don't take effect.
